cli/ballerina-packerina/src/test/java/org/ballerinalang/packerina/cmd/BuildCommandTest.java

```diff
@@ -459,6 +459,18 @@ public void testBuildCommand() throws IOException {
         readOutput(true);
     }
 
+    @Test(description = "Build a valid ballerina file with relative path")
+    public void testBuildWithRelativePath() throws IOException {
+        String buildPath = "relative" + File.separator + "testDir" + File.separator + ".." + File.separator + "testBal"
```

Comment:
Did you check `java.nio.file.Path.relativize` ?

https://stackoverflow.com/questions/37747896/understanding-java-nio-file-path-relativizepath-other

Reply:
This is also fine.